domain/src/main/assets/welcome.json

```diff
@@ -2,7 +2,7 @@
   "author_notes": "",
   "blurb": "",
   "category": "Welcome",
-  "init_state_name": "Welcome!",
+  "init_state_name": "Numeric input",
```

Comment:
This change need to be reverted in before merging with develop and needed only for testing numeric input.

Reply:
There's another numeric input state in this exploration--can we use that, instead?

Reply:
Updated code.